test/lib/ruby-prolog/ruby-prolog_test.rb

```diff
@@ -1,8 +1,7 @@
+#!/usr/bin/env ruby
```

Comment:
What's the purpose of this change?

Reply:
It's my mistake. I fix it! :-)

Sorry!